### PR TITLE
Navigate between 3 pages

### PR DESCRIPTION
**Pull request by wlavery22**
DL's work in iteration 1

### CODE REVIEW

**Review by wlavery22 — COMMENTED** [review]
I've self-reviewed this PR

**Comment by MaryBruff** [comment]
Great! Glad most of the code worked and only needed minor tweaks. Let's start on iteration 2!

File: src/main.js [hunk 1]
```diff
@@ -151,7 +224,4 @@ function showRandomPoster () {
   // 6 show another random poster
   // 7 make your own poster
   // 8 create poster
-  // 9 get random poster /randomizer   
-  
-  //just went through the assignment and looked at the buttons on the example and in the rubric!//
-  //brainstorming!
\ No newline at end of file
+
```
Comment (MaryBruff) [inline src/main.js:227]: Great! Glad most of the code worked and only needed minor tweaks. Let's start on iteration 2!

**Review by MaryBruff — APPROVED** [review]
Great! Glad most of the code worked and only needed minor tweaks. Let's start on iteration 2!

**Pull request merged**